unlock-app/src/components/interface/checkout/Lock.tsx

```diff
@@ -123,8 +123,7 @@ export const TickerSymbol = styled.span`
   font-style: normal;
   font-weight: bold;
   font-size: 20px;
-  margin-left: 4px;
-  text-align: center;
+  text-align: right;
```

Comment:
Once I saw the 2 locks on top of each other, it looked nicer to have the currency symbols aligned